airflow-core/tests/integration/otel/test_otel.py

```diff
@@ -557,6 +557,7 @@ def print_ti_output_for_dag_run(dag_id: str, run_id: str):
 
 @pytest.mark.integration("redis")
 @pytest.mark.backend("postgres")
+@pytest.mark.timeout("240")  # 4 mins timeout for each test.
```

Comment:
4m seems v long. Does it really take that long?

Reply:
Locally, each test takes 1 min. 100 secs should be more than enough. I'll update it.